ui-v2/README.md

```diff
@@ -51,6 +48,9 @@ CONSUL_NODE_CODE=1000
 
 See `./node_modules/@hashicorp/consul-api-double` for more details.
 
+If you wish to run the UI code against a running consul instance, uncomment the `proxy`
+line in `.ember-cli` to point ember-cli to your consul instance.
```

Comment:
I think there's a `--proxy` flag for `ember serve`, would it be worth adding that as a separate script line in `package.json` so then you could do something like `yarn start-with-consul-server` (but a better name) that runs `ember server --proxy http://localhost:3000`?

Reply:
Ah of course yeah! Thanks Matthew, I'll take a look at this before I merge. Btw I don't know if you realized but that other question re: the ember proxy from the other week and your response led me onto this entire PR so thanks for that also 🙌

Reply:
Hey Matthew, I added some more bits on to the end of here, so basically a `make start-consul` target to start against a currently running consul, it uses the `--proxy` flag which is much nicer!

I'll probably merge this tomorrow at some point now this has been added, thanks again!